ts/private/ts_lib.bzl

```diff
@@ -32,10 +32,11 @@ STD_ATTRS = {
     "declaration_dir": attr.string(
         doc = "https://www.typescriptlang.org/tsconfig#declarationDir",
     ),
-    # Note, this is overridden by the @bazel/typescript implementation in build_bazel_rules_nodejs
-    # to add an aspect on the deps attribute.
     "deps": attr.label_list(
-        doc = "Other targets which produce TypeScript typings",
+        doc = """List of labels of other rules that produce TypeScript typings (`.d.ts` files)
```

Comment:
Do we have any standard for the use of the "target" and "label" terms? A label being a name of a target... can a target be referenced by something other then its label such as referencing a pre-declared `.d.ts` output of a target?

Reply:
This change was because the two docstrings for deps were out of sync between the macro and rule... but "labels of other rules" is a mouthful. Just `targets` is better.

Reply:
In this case they have to be targets that provide JsInfo so a reference to a `.d.ts` file won't work; that would have to go in `srcs`